_integration-schemas/slack/v1/users.md

```diff
@@ -0,0 +1,79 @@
+---
+tap: "slack"
+version: "1"
+key: ""
+
+name: "users"
+doc-link: "https://api.slack.com/methods/users.list"
+singer-schema: "https://github.com/singer-io/tap-slack/blob/master/tap_slack/schemas/users.json"
+description: |
+  The `{{ table.name }}` table lists all users in your {{ integration.display_name }} team.
```

Comment:
[From Slack's docs](https://api.slack.com/methods/users.list):

> This method returns a list of all users in the workspace. This includes deleted/deactivated users.

I'd include the bit about deleted and deactivated users. Folks have asked about this in relation to other tables, so if we know the answer to something like this, we should include it.